_episodes_rmd/02-project-intro.Rmd

```diff
@@ -137,3 +156,10 @@ Now we have a good directory structure we will now place/save our data files in
 > We will load and inspect these data later.
 {: .challenge}
 
+### Stage your scripts
+Creating separate R scripts or Rmarkdown documents for different stages of a project will maximise efficiency. 
+For instance, separating data download commands into their own file means that you won't re-download data unneccesarily.
+
+### Save workspaces
```

Comment:
could you please remove this section? We generally discourage saving workspaces because it can lead to hard to find errors, and decreases reproducibility.

Reply:
Will do!